offload/cmake/DetectTestCompiler/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
Shouldn't this be a CMake module instead of a separate project?

Reply:
I simply don't know. It's copied 1:1 from openmp.